README.md

```diff
@@ -80,92 +90,82 @@ Alternatively, you can install it directly from git:
+1. Generate the configuration file with the following command:
 ```bash
 cargo run --release --bin nimiq-client
 ```
-
-### Configuration
-
-By default the client will look for a configuration file in `$HOME/.nimiq/client.toml`.
-In order to create this file yourself, you can use the example config file as follow:
-
+This generates a sample file and places it in a folder `./nimiq`.
+2. Copy the sample configuration file into a new file in the same directory where you will edit it according to your needs:
 ```bash
-cargo run --release --bin nimiq-client                         # Run the client. This will create the example config file.
-cp $HOME/.nimiq/client.example.toml $HOME/.nimiq/client.toml   # Create your config from the example.
-nano $HOME/.nimiq/client.toml                                  # Edit the config. Explanations are included in the file.
+cp $HOME/.nimiq/client.example.toml $HOME/.nimiq/client.toml 
 ```
+3. Edit your configuration file following the explanations inside. Check some [important settings](#configuration) to take into account below.
+4. Run the client:
+```bash
+cargo run --release --bin nimiq-client
+```
+By default, the client will look for the config file in `$HOME/.nimiq/client.toml`.
 
-If you want to directly specify your own configuration file when running the client, you can do so as follow:
-
+**Option B**
+Download the example file and manually place it.
+1. Copy this [sample configuration file](https://github.com/nimiq/core-rs-albatross/blob/albatross/lib/src/config/config_file/client.example.toml) to your preferred location.
+2. Edit the configuration file and adjust settings as needed. Refer to the [configuration settings](#configuration) for guidance.
+3. Run the client with the specified file:
 ```bash
 cargo run --release --bin nimiq-client -- -c path/to/client.toml
 ```
 
-Please take a look at the [`client.example.toml`](lib/src/config/config_file/client.example.toml) for all the configuration options.
+### Service Nodes Guides
+You can also choose to run a validator or a prover node. Check our guides with the full step-by-step description:
+- [Validators](https://www.nimiq.com/developers/build/set-up-your-own-node/becoming-a-validator)
+- [Prover nodes](https://www.nimiq.com/developers/build/set-up-your-own-node/prover-node-guide)
 
-### Testnet
+## Docker
+
+**1.** Create a `data` folder in the main directory with `mkdir ~/data`.
+**2.**  Pull the latest image from the container registry:
+`docker pull ghcr.io/nimiq/core-rs-albatross:latest`.
+**3.** Create a `client.toml` file in `~/data` with `cp ./lib/src/config/config_file/client.example.toml ~/data/client.toml`.
+**4.** Customize the configuration file to match your requirements. Refer to the [sample configuration file](https://github.com/nimiq/core-rs-albatross/blob/albatross/lib/src/config/config_file/client.example.toml) and [configuration settings](#configuration) for guidance.
+**5.**  Run the client via Docker:
```

Comment:
Otherwise the list would not be rendered appropriately:
```suggestion
1. Create a `data` folder in the main directory with `mkdir ~/data`.
2.  Pull the latest image from the container registry:
`docker pull ghcr.io/nimiq/core-rs-albatross:latest`.
3. Create a `client.toml` file in `~/data` with `cp ./lib/src/config/config_file/client.example.toml ~/data/client.toml`.
4. Customize the configuration file to match your requirements. Refer to the [sample configuration file](https://github.com/nimiq/core-rs-albatross/blob/albatross/lib/src/config/config_file/client.example.toml) and [configuration settings](#configuration) for guidance.
5.  Run the client via Docker:
```